soc/arm/nordic_nrf/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 
+zephyr_library()
```

Comment:
nit: to simplify this PR, I'd suggest opening a PR with e879629a70252987bacdbf95162e7587274adcd5

Reply:
Done: #68340.